### PR TITLE
[docs] updated getting started

### PR DESCRIPTION
**Pull request by tanya-rai**
[docs] updated getting started

Cleaned up the getting started as part of docathon - note we will need to do this again when we launch local editor.

### CODE REVIEW

**Review by saqadri — CHANGES_REQUESTED** [review]
Was the typescript code updates tested? Going to pull this branch and make some suggested updates directly

**Comment by tanya-rai** [comment]
There shouldn't be changes to the typescript code - sg on making the suggested updates directly!

File: aiconfig-docs/docs/getting-started.md [hunk 1]
```diff
@@ -304,78 +279,130 @@ await aiConfig.runWithDependencies(
...
 - [CLI Chatbot](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Wizard-GPT)
 - [RAG with AIConfig](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/RAG-with-AIConfig)
-- [Prompt routing](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Basic-Prompt-Routing)
-- [OpenAI function calling](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Function-Calling-OpenAI)
-- [Chain of thought](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Chain-of-Verification)
+- [Prompt Routing](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Basic-Prompt-Routing)
+
+## Coming Soon
+
+- Local editor to iterate on your AIConfigs
```
Comment (rossdanlm) [inline aiconfig-docs/docs/getting-started.md:405]: I thought we'd want to update README to include this?

File: aiconfig-docs/docs/getting-started.md [hunk 2]
```diff
@@ -304,78 +279,130 @@ await aiConfig.runWithDependencies(
...
+- Evaluation interfaces: allow AIConfig artifacts to be evaluated with user-defined eval functions
+- OpenAI Assistants API support
+- Multi-modal ModelParsers: GPT4-V support, Whisper, HuggingFace image generation
```
Comment (rossdanlm) [inline aiconfig-docs/docs/getting-started.md:408]: We already have this! Here are some examples:

- text-to-speech: https://github.com/lastmile-ai/aiconfig/pull/793
- text-to-image (both hugging face and dall-e): https://github.com/lastmile-ai/aiconfig/pull/288, https://github.com/lastmile-ai/aiconfig/pull/460 
- image-to-text: https://github.com/lastmile-ai/aiconfig/pull/821, 
- automatic speech recognition (audio-to-text): https://github.com/lastmile-ai/aiconfig/pull/780

**Review by rossdanlm — APPROVED** [review]
lgtm, just confused that we don't want to advertise our local editor more?

**Pull request closed without merging**